tests/bgv/to_openfhe.mlir

```diff
@@ -0,0 +1,46 @@
+// RUN: heir-opt --bgv-to-openfhe %s > %t
+// RUN: FileCheck %s < %t
+
+// This simply tests for syntax.
+
+#my_poly = #polynomial.polynomial<1 + x**1024>
+#ring1 = #polynomial.ring<cmod=463187969, ideal=#my_poly>
+#ring2 = #polynomial.ring<cmod=33538049, ideal=#my_poly>
+#rings = #bgv.rings<#ring1, #ring2>
+#rings2 = #bgv.rings<#ring1, #ring2, #ring2>
+!ct = !bgv.ciphertext<rings=#rings, dim=2, level=1>
+!ct_dim = !bgv.ciphertext<rings=#rings, dim=4, level=1>
+!ct_level = !bgv.ciphertext<rings=#rings2, dim=2, level=2>
+
+// CHECK: module
+module {
+  // CHECK: func.func @test_fn([[X:%.+]]: [[T:.*33538049.*]]) -> [[T]] {
+  func.func @test_fn(%x : !ct) -> !ct {
+    // CHECK: return [[X]] : [[T]]
+    return %x : !ct
+  }
+
+  // CHECK: func.func @test_ops([[C:%.+]]: [[S:.*crypto_context.*]], [[X:%.+]]: [[T:.*33538049.*]], [[Y:%.+]]: [[T]]) {
+  func.func @test_ops(%x : !ct, %y : !ct) {
+    %negate = bgv.negate(%x) : !ct
```

Comment:
Please add `// CHECK` directives for what ops these should lower to.

Reply:
Applied!